international-labour-organization.csl

```diff
@@ -190,17 +192,24 @@
         <else-if type="article-journal">
           <text macro="title" prefix=" &quot;" suffix="&quot;,"/>
           <group prefix=" " delimiter=" ">
-            <text term="in" suffix=":"/>
+            <text term="in"/>
             <text macro="editor"/>
             <text variable="container-title" font-style="italic" suffix=","/>
             <text macro="edition"/>
             <group suffix="." delimiter=", ">
               <text macro="publisher"/>
-              <text variable="volume"/>
+              <text variable="volume" text-case="capitalize-first"/>
               <text macro="pages"/>
             </group>
           </group>
         </else-if>
+        <else-if type="article-newspaper" match="any">
+          <group>
+            <text macro="title" prefix=" &quot;" suffix=".&quot;"/>
```

Comment:
It's better to use the "quotes" attribute here (`quotes="true"`). See http://docs.citationstyles.org/en/stable/specification.html#quotes